### PR TITLE
Create cluster role to allow access eks console resources

### PR DESCRIPTION
**Pull request by yuriadams**

### CODE REVIEW

**Comment by yuriadams** [comment]
https://convox602-my.sharepoint.com/:w:/g/personal/yuri_adams_convox_com/ETz7Yx0JNxJCmsy0Jz0_6BwBbMXcLp0HIEoX82nARiQFkg?e=hhIwZg

**Comment by Twsouza** [comment]
@yuriadams where are you going to publish the doc?

**Comment by yuriadams** [comment]
Good question, @Twsouza. Do you have a suggestion?

File: docs/management/console-rack-management.md [hunk 1]
```diff
@@ -59,3 +59,65 @@ You can move any Console-managed Rack back to being locally managed only with th
     staging            gcp       running
 ```
 Terraform state will be transferred to your local machine for exclusive management.
+## View Kubernetes resources
+You can view the Kubernetes resources deployed to your cluster with the AWS Management Console.  
+
+### Steps:
+- Create a Kubernetes clusterrolebinding that is bound to a Kubernetes clusterrole that has the necessary permissions to view the Kubernetes resources. To learn more about Kubernetes roles and role bindings, see Using RBAC Authorization in the Kubernetes documentation. You can apply one of the following manifests to your cluster that create a role and rolebinding or a clusterrole and clusterrolebinding with the necessary Kubernetes permissions: *
+If your rack version is > 3.6.4, you don’t need to create the Cluster role and the Cluster role binding.
```
Comment (Twsouza) [inline docs/management/console-rack-management.md:67]: Can you move the _You can skip it for rack greater or equal to 3.6.4_ to the begninng? using Pascal case or separating the words on `clusterrolebinding` will be easier to read? (i.e. `ClusterRoleBiding` or _cluster role binding_)

Comment (Twsouza) [reply docs/management/console-rack-management.md:67]: > Using RBAC Authorization in the Kubernetes documentation

Can you link the referred documentation here?

File: docs/management/console-rack-management.md [hunk 2]
```diff
@@ -59,3 +59,65 @@ You can move any Console-managed Rack back to being locally managed only with th
...
+        },
+        {
+            "Effect": "Allow",
+            "Action": "ssm:GetParameter",
+            "Resource": "arn:aws:ssm:*:111122223333:parameter/*"
+        }
+    ]
+}   
+```
+- For more information about adding users or roles to the aws-auth ConfigMap, see [Add IAM users, roles, or AWS accounts to the ConfigMap](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html#aws-auth-users).
```
Comment (Twsouza) [inline docs/management/console-rack-management.md:102]: Can you add a warning here that misconfiguring the `aws-auth` can lock the user out of their rack and they must proceed with caution?

File: docs/management/console-rack-management.md [hunk 3]
```diff
@@ -59,3 +59,65 @@ You can move any Console-managed Rack back to being locally managed only with th
     staging            gcp       running
 ```
 Terraform state will be transferred to your local machine for exclusive management.
+## View Kubernetes resources
```
Comment (Twsouza) [inline docs/management/console-rack-management.md:62]:
```suggestion
## Access rack cluster in the EKS UI (AWS only)
```

Specify that this is only applicable to AWS.

**Review by Twsouza — CHANGES_REQUESTED** [review]

File: docs/management/console-rack-management.md (hunk 2 above)
Comment (yuriadams) [reply docs/management/console-rack-management.md:102]: sure!

**Review by heronrs — APPROVED** [review]

**Review by Twsouza — APPROVED** [review]

**Pull request merged**